config/locales/devise.en.yml

```diff
@@ -10,7 +10,7 @@ en:
       already_authenticated: ""
       inactive: "Your account is not activated yet."
       invalid: "Invalid %{authentication_keys} or password."
-      locked: "Your account is locked."
+      locked: "Your account has been locked. Reset your password"
```

Comment:
This messaging is probably wrong now we're not guiding people to resetting their password.

Not sure what the right messaging is - "Your account is locked." might be a little unhelpful, but "Your account is locked, check your e-mail" gives a hint to the attacker that they can get access by e-mail.